src/Pods/Integrations/WPML.php

```diff
@@ -0,0 +1,335 @@
+								AND `wpml_translations`.`language_code` = '" . pods_sanitize( $current_language ) . "'
+					";
+				}
+				break;
+
+			case 'taxonomy':
+			case 'term':
+			case 'nav_menu':
+			case 'post_format':
+				if ( $this->is_translated_taxonomy( $object_name ) ) {
+					$wpml_translations = "
+						LEFT JOIN `{$db_prefix}icl_translations` AS `wpml_translations`
+							ON `wpml_translations`.`element_id` = `tt`.`term_taxonomy_id`
+								AND `wpml_translations`.`element_type` = 'tax_" . pods_sanitize( $object_name ) . "'
+								AND `wpml_translations`.`language_code` = '" . pods_sanitize( $current_language ) . "'
+					";
+				}
+				break;
+		}
+
+		if ( $wpml_translations ) {
+
+			$info['join']['wpml_translations'] = $wpml_translations;
+
+			$info['join']['wpml_languages'] = "
+				LEFT JOIN `{$db_prefix}icl_languages` AS `wpml_languages`
+					ON `wpml_languages`.`code` = `wpml_translations`.`language_code` AND `wpml_languages`.`active` = 1
+			";
+
+			$info['where']['wpml_languages'] = "`wpml_languages`.`code` IS NOT NULL";
+		}
+
+		return $info;
+	}
+
+	/**
+	 * @param array $data
+	 *
+	 * @return array
+	 */
+	public function pods_component_i18n_admin_data( $data ) {
+
+		foreach ( $data as $lang => $field_data ) {
+			if ( in_array( $lang, $this->get_locales(), true ) ) {
+				$data[ $lang ]['wpml'] = true;
+			} else {
+				$data[ $lang ]['wpml'] = false;
+			}
+		}
+
+		return $data;
+	}
+
+	/**
+	 * @param array $fields
+	 * @param array $data
+	 *
+	 * @return array
+	 */
+	public function pods_component_i18n_admin_ui_fields( $fields, $data ) {
+
+		$fields['manage']['wpml'] = array(
+			'label' => __( 'WPML', 'pods' ),
+			'type'  => 'boolean',
+		);
+
+		return $fields;
+	}
+
+	/**
+	 * Helper method for backwards compatibility.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @param string $object_name
+	 *
+	 * @return false|mixed|void
+	 */
+	public function is_translated_post_type( $object_name ) {
+		global $sitepress;
+		if ( has_filter( 'wpml_is_translated_post_type' ) ) {
+			return apply_filters( 'wpml_is_translated_post_type', false, $object_name );
+		} elseif ( is_callable( [ $sitepress, 'is_translated_post_type' ] ) ) {
+			return $sitepress->is_translated_post_type( $object_name );
+		}
+		return false;
+	}
+
+	/**
+	 * Helper method for backwards compatibility.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @param string $object_name
+	 *
+	 * @return false|mixed|void
+	 */
+	public function is_translated_taxonomy( $object_name ) {
+		global $sitepress;
+		if ( has_filter( 'wpml_is_translated_taxonomy' ) ) {
+			return apply_filters( 'wpml_is_translated_taxonomy', false, $object_name );
+		} elseif ( is_callable( [ $sitepress, 'is_translated_taxonomy' ] ) ) {
+			return $sitepress->is_translated_taxonomy( $object_name );
+		}
+		return false;
+	}
+
+	/**
+	 * @return array
+	 */
+	public function get_language( $locale ) {
+		$languages = apply_filters( 'wpml_active_languages', array() );
```

Comment:
Should this call $this->get_languages()?

Reply:
Good find!